go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/projectcontour/contour
 
-go 1.16
+go 1.19
```

Comment:
CONTRIBUTING.md has a specific ref to Go 1.16, might want to just make that say to look in this file for the minimum Go version

Reply:
Ah good call